packages/core/admin/admin/src/components/GuidedTour/layout.js

```diff
@@ -0,0 +1,153 @@
+const layout = {
+  contentTypeBuilder: {
+    home: {
+      title: {
+        id: 'app.components.GuidedTour.home.CTB.title',
+        defaultMessage: '🧠 Build the content structure',
+      },
+      cta: {
+        title: {
+          id: 'app.components.GuidedTour.home.CTB.cta.title',
+          defaultMessage: 'Go to the Content-type Builder',
+        },
+        type: 'REDIRECT',
+        target: '/plugins/content-type-builder',
+      },
+    },
+    create: {
+      title: {
+        id: 'app.components.GuidedTour.CTB.create.title',
+        defaultMessage: '🧠 Create a first Collection-type',
+      },
+      content: {
+        id: 'app.components.GuidedTour.CTB.create.content',
+        defaultMessage:
+          '<p>Collection-types help you manage several entries, Single-types are suitable to manage only one entry.</p> <p>Ex: For a website, articles would be a Collection type and homepage would be a Single type.</p>',
```

Comment:
```suggestion
          '<p>Collection-types help you manage several entries, Single-types are suitable to manage only one entry.</p> <p>Ex: For a website, articles would be a Collection-type where as homepage would be a Single-type.</p>',
```

Reply:
product copy - @iparidhi @maevalienard

Reply:
For Collection type and Single type, it has been decided internally to use these terms without the dash, so I'd rather stick to that decision.

As for the where as, I don't have an opinion :D

Reply:
I checked with Grammarly, it tells me that **_whereas a homepage_** would be better, WDYT?

Reply:
@ronronscelestes I think you are right. Might be worth checking with a native speaker though ...

Reply:
@gu-stav indeed!

Reply:
Hi @derrickmehaffy, could you help us with this topic? 😄